corehq/apps/export/forms.py

```diff
@@ -1,139 +1,336 @@
-        self.helper.form_class = "form-horizontal"
+        self.helper.form_tag = False
+        self.helper.label_class = 'col-sm-2'
+        self.helper.field_class = 'col-sm-10'
 
         self.helper.layout = crispy.Layout(
-            crispy.Fieldset(
-                _('Select Form'),
-                crispy.Field(
-                    'application',
-                    data_bind='value: appId',
-                ),
-                crispy.Div(
-                    crispy.Field(
-                        'module',
-                        data_bind=(
-                            "options: moduleOptions, "
-                            "optionsText: 'text', "
-                            "optionsValue: 'value', "
-                            "value: moduleId"
-                        ),
-                    ),
-                    data_bind="visible: appId",
-                ),
-                crispy.Div(
-                    crispy.Field(
-                        'form',
-                        data_bind=(
-                            "options: formOptions, "
-                            "optionsText: 'text', "
-                            "optionsValue: 'value', "
-                            "value: formId"
-                        ),
-                    ),
-                    data_bind="visible: moduleId",
-                ),
+            crispy.Field(
+                'application',
+                placeholder=_("Select Application"),
+                ng_model="createForm.application",
+                ng_change="updateModules()",
+                ng_required="true",
             ),
-            crispy.Div(
-                FormActions(
-                    crispy.ButtonHolder(
-                        crispy.Submit(
-                            'create_export',
-                            _('Next'),
-                        ),
-                    ),
-                ),
-                data_bind="visible: formId",
+            crispy.Field(
+                'module',
+                placeholder=_("Select Module"),
+                ng_model="createForm.module",
+                ng_disabled="!createForm.application",
+                ng_change="updateForms()",
+                ng_required="true",
+            ),
+            crispy.Field(
+                'form',
+                placeholder=_("Select Form"),
+                ng_model="createForm.form",
+                ng_disabled="!createForm.module",
+                ng_required="true",
             ),
         )
 
 
-class CreateCaseExportForm(forms.Form):
-    application = forms.ChoiceField()
-    case_type = forms.ChoiceField()
+class CreateCaseExportTagForm(forms.Form):
+    """The information necessary to create an export tag to begin creating a
+    Case Export. This form interacts with CreateExportController in
+    list_exports.ng.js
+    """
+    application = forms.CharField(required=False)
+    case_type = forms.CharField(
+        required=False,
```

Comment:
probably my own ignorance, but curious why form and case_type aren't required in these forms